dev/ok/helmfile.yaml

```diff
@@ -0,0 +1,48 @@
+# single env example file
+# helm file dependencies
+# helm plugin install https://github.com/databus23/helm-diff
+# repositories to be installed
+repositories:
+  - name: datarepo-helm
+    url: https://broadinstitute.github.io/datarepo-helm
+
+# helm releases to be deployed
+releases:
+  - name: ok-jade-create-secret-manager-secret    # release name
+    namespace: ok   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/create-secret-manager-secret   # chart name
+    version: 0.0.6   #    Chart version
+    missingFileHandler: Warn
+    values:
+      - create-secret-manager-secret.yaml      # Value files passed via --values
+  - name: ok-jade-gcloud-sqlproxy    # release name
+    namespace: ok   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/gcloud-sqlproxy   # chart name
+    version: 0.19.7 #    Chart version
+    missingFileHandler: Warn
+    values:
+      - gcloud-sqlproxy.yaml    # Value files passed via --values
+  - name: ok-jade-datarepo-api   # name of this release
+    namespace: ok   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/datarepo-api   # the chart name
+    missingFileHandler: Warn
+    values:
+      - datarepo-api.yaml   # Value files passed via --values
+  - name: ok-jade-datarepo-ui   # name of this release
+    namespace: ok   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/datarepo-ui   # the chart name
+    missingFileHandler: Warn
+    values:
+      - datarepo-ui.yaml   # Value files passed via --values
+  - name: ok-jade-oidc-proxy   # name of this release
+    namespace: ok   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/oidc-proxy   # the chart name
+    version: 0.0.25    # chart version
```

Comment:
latest = 0.0.31
https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo/Chart.lock#L7

Reply:
Updated.  I can make a change to the onboarding docs indicating that we should check our new helmfile against the latest chart versions.  Is the reason that `ok-jade-datarepo-api` and `ok-jade-datarepo-ui` don't have versions specified here so that they always pick up latest?

Reply:
Perfect and exactly!